Alphabetical order of custom widget icons #4664

ashklianko opened the pull request:
-Moved toolbar classes to the separate files
-Sorting non default apps alphabetically

ashklianko committed:
Alphabetical order of custom widget icons #4664

Commit:
update failed ui-tests in branch 4511

alansemenov commented on modules/lib/src/main/resources/assets/js/app/widget/WidgetsToolbar.ts:
```diff
@@ -0,0 +1,160 @@
+            }
+
+            // finding
+            if (newButton.getWidgetDisplayName().toLowerCase().localeCompare(existingButton.getWidgetDisplayName().toLowerCase()) < 0) {
```
`newButton.getWidgetDisplayName().toLowerCase()` should be called once before `.some()`. Actually, we don't even need entire `newButton` in this function, we only need its display name.

Commit:
ui-tests - update id of button in sidebar

ashklianko committed:
Pasting text from Google Docs includes underline for links #4687

-Something changed rather in google docs or in CKEeditor so paste data is slightly changed and we couldn't track that paste was from google doc

alansemenov committed:
Merge branch 'master' into issue-4664
